src/lunamary/ReadWriteData/file.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
Please, move these kinds of files to a packaga named resources under you directory name.

Reply:
Done!